### DISCUSSION TITLE
Rdio Scanner App not working

### DISCUSSION THREAD

**Discussion by Egr1791** [chuot/rdio-scanner#444] [General]
Has the Rdio Scanner App stopped providing live feeds for my local police department and fire department, etc. it seems to have stopped working all together?

**Comment by ScannerGrid** [comment]
Rdio is a platform that serves as the software used to deliver audio streams provided by volunteers in specific regions. It's important to note that Rdio itself does not host or manage the feeds directly; it simply facilitates the broadcasting of audio from local contributors. If you are experiencing issues with a particular scanner feed, it’s a good idea to contact the individual or organization who originally shared the Rdio scanner link. They are likely the ones responsible for maintaining and hosting those feeds, and they may be able to provide insight into whether the feed is temporarily down or has been permanently discontinued. Volunteers play a key role in keeping these feeds active, so their availability often determines the status of the audio streams for a given area
